Postgres/MySQL/MSSQL: Add setting to limit the maximum number of rows processed (#38986)
grafana/grafana: docs/sources/administration/configuration.md

If enabled and user is not anonymous, data proxy will add X-Grafana-User header with username into the request. Default is `false`.

### row_limit

> Limit the amount of rows being processed from data sources that returns rows, e.g. SQL data sources. Default is `1000000`.

Reviewer: ```suggestion
Limits the number of rows that Grafana will process from SQL (relational) data sources.  Default is `1000000`.
```
